src/networking.c

```diff
@@ -2760,15 +2764,19 @@ void processMultibulkBuffer(client *c) {
              * just use the current sds string. */
             if (!is_primary && c->qb_pos == 0 && c->bulklen >= PROTO_MBULK_BIG_ARG &&
                 sdslen(c->querybuf) == (size_t)(c->bulklen + 2)) {
-                c->argv[c->argc++] = createObject(OBJ_STRING, c->querybuf);
+                robj *o = createObject(OBJ_STRING, c->querybuf);
+                o->allocator_id = getCurThreadID();
```

Comment:
Is `c->querybuf` always allocated by the same thread as the robj, the current thread here?